### DISCUSSION TITLE
PrimeVue 4 & Templates -- upgrade or use Aura?

### DISCUSSION THREAD

**Discussion by bitterloa** [primefaces/community#1604] [PrimeVue Templates] [answered]
Regarding purchased layouts/presets and the newest PrimeVue v4 -- I just downloaded the latest update to Freya Layout v5.2.0 released a couple weeks ago and noticed it is updated to work with PrimeVue 3.49.1 - will purchased themes be updated to PrimeVue v4? Or, if I want to upgrade a project I'm working on to v4 should I completely ditch the paid themes (i.e. Freya, Atlantis, etc) and use the new Aura preset (or Unstyled with Tailwind)? 

From my research it looks like moving from PrimeVue 3 to 4 that perhaps the paid themes should not be used as the underlying architecture for CSS has been completely redone. The changes look exciting but what is the best approach to achieve the same functionality with sidebar menus, layout etc. in v4 -- should we wait for a soon to be released update to paid themes or plan on starting from scratch using Aura or unstyled Tailwind? I need to know as I'm launching a new app soon and ideally want to upgrade to v4 before launch. Thanks!

p.s. I asked this question already on Discord channel but did not get a response. 

![image](https://github.com/primefaces/community/assets/1977439/f67a3941-6738-42a9-939d-5d1c0bc87b4d)

**Comment by cantin-f** [comment]
Same question for me, i'm stuck with an old version of the template waiting for a roadmap. I don't want to work on latest template version that already looks obsolete.

**Comment by tugcekucukoglu** [comment, marked as answer]
Currently, only version 4.0.0.beta-1 has been released. That's why everything is new and not stable. But when the stable version is released, all templates will be updated with v4.

**Comment by basitiyidir** [comment]
The question was asked on April 4th and three months have passed. However, there has been no progress in updating the templates. They say all templates will be updated in Q3. PrimeVue Roadmap. https://primevue.org/roadmap/

However, the Prime team works a bit slowly. Frankly, I don't think the templates will be updated to PrimeVue v4 by the end of Q3.

Therefore, although we follow the developments in PrimeVue v4 with excitement, we are stuck with PrimeVue v3 for current and new projects. Starting and designing a template from scratch in PrimeVue v4 is quite laborious for us.
